receiver/googlecloudspannerreceiver/factory.go

```diff
@@ -31,9 +31,7 @@ func NewFactory() receiver.Factory {
 
 func createDefaultConfig() component.Config {
 	return &Config{
-		ScraperControllerSettings: scraperhelper.ScraperControllerSettings{
-			CollectionInterval: defaultCollectionInterval,
-		},
+		ScraperControllerSettings:         scraperhelper.NewDefaultScraperControllerSettings(metadata.Type),
```

Comment:
The `defaultCollectionInterval` const defined above is now left unused. This could trick someone trying to change it and seeing no effect. We should remove that const.